ports/esp32/modnetwork.c

```diff
@@ -229,7 +229,7 @@ STATIC const mp_rom_map_elem_t mp_module_network_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_WLAN), MP_ROM_PTR(&get_wlan_obj) },
     #endif
 
-    #if (ESP_IDF_VERSION_MAJOR == 4) && (ESP_IDF_VERSION_MINOR >= 1) && (CONFIG_IDF_TARGET_ESP32)
+    #if MODNETWORK_LAN
```

Comment:
This config macro should match others and be `MICROPY_PY_NETWORK_LAN`.

Reply:
Done.